solarwinds_apm/configurator.py

```diff
@@ -559,19 +573,30 @@ def _add_all_instrumented_python_framework_versions(
 
             # Set up Instrumented Library Versions KVs with several special cases
             entry_point_name = entry_point.name
-            # Some OTel instrumentation libraries are named not exactly
-            # the same as the instrumented libraries!
+            # Some OTel instrumentation library entry point names are not exactly
+            # the same as their corresponding instrumented libraries
             # https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/README.md
-            if entry_point_name == "aiohttp_client":
+            if entry_point_name == "aiohttp-client":
                 entry_point_name = "aiohttp"
+            # Both client/server instrumentors instrument `aiohttp`
+            # so key is potentially overwritten, not duplicated
+            elif entry_point_name == "aiohttp-server":
+                entry_point_name = "aiohttp"
+            elif entry_point_name == "aio-pika":
```

Comment:
This might only be a problem at configurator and not exporter https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/instrumentation/opentelemetry-instrumentation-aio-pika/pyproject.toml#L38